protogen/build.gradle

```diff
@@ -1,5 +1,10 @@
 dependencies {
     compile project(':conductor-common')
-    compile 'com.squareup:javapoet:1.11.1'
-    compile 'com.github.jknack:handlebars:4.0.6'
+    compile "com.github.vmg.protogen:protogen-codegen:${revProtoGen}"
+}
+
+task generate(type: JavaExec) {
```

Comment:
What I envision is to have a protogen Gradle plugin that scans classes annotated with ProtoMessage and generates proto definitions and mappers automatically, it should be configured to run before generateProto task (from protobuf-gradle-plugin), we don't have to do this now, but at least we should probably get rid of ConductorProtoGen and have a main class in vmg.protogen that uses Reflections to get classes annotated with @ProtoMessage within a package (passed in as an argument) to process.

Reply:
This was exceedingly difficult to get right. See: https://github.com/Netflix/conductor/pull/636/commits/46ed3b17856a691897a6067387a5df5664d266ce